.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please exclude IDE files from commits

Reply:
IDE files removed from commit. Include .vscode/ in gitignore